library/core/src/slice/mod.rs

```diff
@@ -2031,6 +2034,256 @@ impl<T> [T] {
         unsafe { (from_raw_parts_mut(ptr, mid), from_raw_parts_mut(ptr.add(mid), len - mid)) }
     }
 
+    /// Divides one slice into two at an index returning, `None` if slice is too
+    /// short.
```

Comment:
The comma is misplaced.

```suggestion
    /// Divides one slice into two at an index, returning `None` if the slice is too
    /// short.
```